buildpipeline/pipelines.json

```diff
@@ -141,6 +141,21 @@
             "Architecture": "arm",
             "PB_BuildType": null
           }
+        },
+        {
+          "Name": "DotNet-CoreClr-Trusted-Linux-Crossbuild-arm64",
```

Comment:
This should be DotNet-CoreClr-Trusted-Linux-Crossbuild, otherwise you'd need to add DotNet-CoreClr-Trusted-Linux-Crossbuild-arm64.json file too. Which seems not to be needed since the existing DotNet-CoreClr-Trusted-Linux-Crossbuild.json is fully parametrized by the architecture.